fixes for workflows doc (#1959)
quiltdata/quilt: docs/Advanced Features/workflows.md

QuiltException: Metadata failed validation: 'superhero' is a required property.
```

We encountered another exception because the `beta` workflow specifies
`metadata_schema: superheroes`.

> Therefore, the `test/package` metadata must be validated against the

Reviewer: @akarve "must validate" or "must be validated"? which one sounds better?

> Therefore, the `test/package` metadata must be validated against the

Reviewer 2: ```suggestion
Therefore, the `test/package` metadata must validate against the
```

Package test/package@06b2815 pushed to s3://quilt-sergey-dev-metadata
```

> Also `default_workflow` can be set in the config to specify which workflow will be used

Reviewer 2: ```suggestion
You can also specify a `default_workflow` in `config.yml`.
```

Author: I left this up to you, but generally I'd like to be less specific here (we already specified path in the beginning) and do not mention `config.yml` again since we could change the path in the future or maybe even move this configuration out of S3.
